### PR TITLE
Add `ROOT File Viewer` blog post

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
Hi everyone!

As suggested by @Axel-Naumann, here's a blog post proposal introducing the [ROOT File Viewer](https://marketplace.visualstudio.com/items?itemName=albertopdrf.root-file-viewer) VS Code extension! I think I didn't miss anything, but will gladly make any necessary changes. Also, recommendations on how to improve it will be greatly appreciated. Besides, I've set its date to tomorrow, but it can be changed to what you think it's best. Let me know what you think of it! :slightly_smiling_face:

### CODE REVIEW

**Comment by eguiraud** [comment]
@couet what's the right way to make @AlbertoPdRF appear as the author, as we did for the author of https://root.cern/blog/snap-announcement/ ?

File: blog/_posts/2021-03-11-vscode-extension-announcement.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+- Launching VS Code's `Quick Open` (`Ctrl + P`), pasting `ext install albertopdrf.root-file-viewer`, and pressing enter
+- or searching `ROOT File Viewer` directly within VS Code's `Extensions` view (`Ctrl + Shift + X` or click on the `Extensions` icon in the `Activity Bar`)
+- or running `code --install-extension albertopdrf.root-file-viewer` from the command line
+- or... well, many other options!
+
+And if you want to know more, just keep reading!
+
+## Visual Studio Code
+
+VS Code is a free, [open source](https://github.com/Microsoft/vscode/), and very popular code editor by developed by Microsoft with TypeScript, a superset of JavaScript. It runs on any operating system, supports many languages, has built in support for Git, and much more. Moreover, its functionality can easily be extended thanks to the [Extension API](https://code.visualstudio.com/api). This is where the fun begins!
```
Comment (eguiraud) [inline blog/_posts/2021-03-11-vscode-extension-announcement.md:30]:
```suggestion
VS Code is a free, [open source](https://github.com/Microsoft/vscode/), and very popular code editor developed by Microsoft with TypeScript, a superset of JavaScript. It runs on any operating system, supports many languages, has built-in support for Git, and much more. Moreover, its functionality can easily be extended thanks to the [Extension API](https://code.visualstudio.com/api). This is where the fun begins!
```

**Review by eguiraud — APPROVED** [review]
LGTM! and congratulations @AlbertoPdRF on the amazing work on the extension

**Comment by AlbertoPdRF** [comment]
Thanks for the kind words and the blazing fast review @eguiraud!

**Comment by Axel-Naumann** [comment]
I have a couple of minor comments, but I'd also like to see the preview at https://root.cern/524/ - but that cannot be generated because I caused tons of broken links (@bellenot @couet are on it). I.e. we'll fix the links, then generate the preview for this PR, then I submit my review :-) Apologies this is so involved this time!

**Comment by couet** [comment]
I have fixed the broken links in an other PR. The new PR builds without mistakes. So the broken links are fixed. The web site is build without the broken links. But when I relaunch the  build of this one it still fails...It seems this PR does not see the fixes I did. Is there something to do in order this PR sees the fixes I did in the other ? a kind of rebase ? ....

**Comment by AlbertoPdRF** [comment]
> I have fixed the broken links in an other PR. The new PR builds without mistakes. So the broken links are fixed. The web site is build without the broken links. But when I relaunch the build of this one it still fails...It seems this PR does not see the fixes I did. Is there something to do in order this PR sees the fixes I did in the other ? a kind of rebase ? ....

It probably needs a rebase, yeah, I'll do it now!

**Comment by couet** [comment]
Yep, it works. Let me merge .

**Comment by eguiraud** [comment]
@couet I think Axel wanted to review?

**Comment by AlbertoPdRF** [comment]
I can submit a follow up PR with his suggestions when he gets to review it

**Comment by couet** [comment]
Ah ok ... I was too fast. We can edit it if something need to be changed. It looks very good to me.

**Review by Axel-Naumann — COMMENTED** [review]
@couet - as I said, I had a tiny comment :-(

**Comment by Axel-Naumann** [comment]
@couet I'll take care of this comment myself.

**Comment by Axel-Naumann** [comment]
@AlbertoPdRF I think it'd be super nice if you could increase the font size of VSCode in the GIF, such that the folders on the left are visible?

**Comment by AlbertoPdRF** [comment]
> @AlbertoPdRF I think it'd be super nice if you could increase the font size of VSCode in the GIF, such that the folders on the left are visible?

I'll try when I'm back home!

**Comment by Axel-Naumann** [comment]
Wording addressed in https://github.com/root-project/web/pull/525/commits/836032f5e8d4d6dd7722dc2664e781e02c86a335

**Comment by AlbertoPdRF** [comment]
> > @AlbertoPdRF I think it'd be super nice if you could increase the font size of VSCode in the GIF, such that the folders on the left are visible?
> 
> I'll try when I'm back home!

JFTR: this is addressed in #527

**Pull request merged**